Apps/phsplunkitsi/README.md

```diff
@@ -4,7 +4,7 @@ Author: Splunk Inc.
 Latest Version Tag: 1.0.0
```

Comment:
Should be 1.0.1

Reply:
Made the suggested change in the latest commit: https://github.com/phantomcyber/phantom-apps/pull/356/commits/a9899812a5c57887de35ac0da37e5f1d6b576aa3